kano_updater/scenarios.py

```diff
@@ -359,4 +359,22 @@ def beta_210_to_beta_220(self):
             )
 
     def beta_220_to_beta_230(self):
-        pass
+        os.system('sudo adduser {} i2c'.format(get_user_unsudoed()))
```

Comment:
It looks like `kano-greeter` and `kano-init` use something similar [here](https://github.com/KanoComputing/kano-init/blob/master/kano_init/user.py#L141) and [here](https://github.com/KanoComputing/kano-greeter/blob/master/bin/kano-greeter-account#L96). Good catch, will have it changed.